core_fw/ipc_radio/Kconfig

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+mainmenu "Nordic ipc_radio firmware"
+
+config IPC_RADIO_802154
+	bool "IPC IEEE 802.15.4 radio"
+	select NRF_802154_SER_RADIO
+	default n
+	help
+	  Enable the IPC IEEE 802.15.4 radio serialization.
+
+config IPC_RADIO_BT
+	bool "IPC Bluetooth"
+	default n
+	help
+	  Enable the IPC Bluetooth radio serialization.
+
+if IPC_RADIO_BT
+
+choice IPC_RADIO_BT_SER
+	default IPC_RADIO_BT_HCI_IPC
+	prompt "Bluetooth serialization type"
+	help
+	  Type of the IPC Bluetooth radio serialization used.
+
+config IPC_RADIO_BT_HCI_IPC
+	bool "Bluetooth HCI serialization"
+	depends on IPC_SERVICE
+	depends on BT_HCI_RAW
+	help
+	  Use Bluetooth HCI serialization over ipc_service.
+
+config IPC_RADIO_BT_RPC
+	bool "Bluetooth host API serialization over RPC"
+	depends on BT_RPC_HOST
+	help
+	  Use nRF RPC serialization for Bluetooth.
+
+endchoice # IPC_RADIO_BT_SER
+
+config BT_MAX_CONN
```

Comment:
Could it be in specific conf overlay files?

Reply:
The conf files can override this value. The default are set to ensure the default values as specified by this comment https://github.com/nrfconnect/sdk-nrf/pull/12588#discussion_r1355061245